appeals/web/src/server/appeals/appeal-details/hearing/setup/__tests__/set-up-hearing.test.js

```diff
@@ -0,0 +1,217 @@
+// @ts-nocheck
+import { appealData } from '#testing/app/fixtures/referencedata.js';
+import { createTestEnvironment } from '#testing/index.js';
+import { parseHtml } from '@pins/platform';
+import nock from 'nock';
+import supertest from 'supertest';
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+const baseUrl = '/appeals-service/appeal-details';
+
+describe('set up hearing', () => {
+	beforeEach(installMockApi);
+	afterEach(teardown);
+
+	describe('GET /hearing/setup', () => {
+		// eslint-disable-next-line jest/expect-expect
+		it('should redirect to /hearing/setup/date', () => {
+			return new Promise((resolve) => {
+				request
+					.get(`${baseUrl}/2/hearing/setup`)
+					.expect(302)
+					.expect('Location', `${baseUrl}/2/hearing/setup/date`)
+					.end(resolve);
+			});
+		});
+	});
+
+	describe('GET /hearing/setup/date', () => {
+		const appealId = 2;
+
+		/** @type {*} */
```

Comment:
It should be possible to remove this given the `// @ts-nocheck` at the top of the file, it's a trivial issue though